src/Factory.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Friendica;
+
+use Psr\Log\LoggerInterface;
+
+/**
+ * Factories act as an intermediary to avoid direct Entitiy instanciation.
+ *
+ * @see Model
+ * @see Collection
+ */
+abstract class Factory
```

Comment:
I'm sorry to start this discussion again, but one thing is really nasty about having classes outside of their namespace:

If you `use Friendica\Factory;` inside class, it isn't clear if you mean the namespace or the abstract class itself. I get all the time "duplicate" notices in PHPStorm ..

Reply:
So, back to:
- Friendica\BaseFactory
- Friendica\BaseModel
- Friendica\BaseRepository
- Friendica\BaseCollection

?

Reply:
yep .. I guess so